docs/fsharp/language-reference/parameters-and-arguments.md

```diff
@@ -122,7 +120,14 @@ w.Height // = 10
 
 Note that those members could perform any arbitrary work, the syntax is effectively a short hand to call property setters before returning the final value.
 
-## Optional Parameters
+## Optional parameters
+
+F# supports two distinct forms of optional parameters for methods, each serving different purposes:
+
+- [Optional arguments with late defaults (F# native)](#optional-arguments-with-late-defaults-f-native)
+- [Optional arguments with early defaults (C# interop)](#optional-arguments-with-early-defaults-c-interop)
+
+### Optional arguments with late defaults (F# native)
 
 You can specify an optional parameter for a method by using a question mark in front of the parameter name. From the callee's perspective, optional parameters are interpreted as the F# option type, so you can query them in the regular way that option types are queried, by using a `match` expression with `Some` and `None`. Optional parameters are permitted only on members, not on functions created by using `let` bindings.
```

Comment:
Should also mention usage of `defaultArg` and shadowing to convert an optional argument to a value.
Possibly also mention the advantage - unlike "C# style extensions", this allows authors of the method to tell if caller passed in a value or not.